web_app/hazard-reporting-map/styles/globals.css

```diff
@@ -34,4 +42,30 @@ body {
 
 .mapboxgl-popup-content {
   background: transparent !important;
-}
\ No newline at end of file
+}
+table {
```

Comment:
dont use general tags to update the css otherwise it will apply to all the table th td tags that will be used in other components and overwrite the styles.

Use tailwind inline css for these designs

Reply:
Done

Reply:
all the issues fixed let's wait for the final design then we will merge according to the final design

Reply:
I've updated the design.